app/views/petitions/_closed_petition_show.html.erb

```diff
@@ -16,7 +16,27 @@
   <span>All petitions run for 6 months</span>
 </p>
 
-<p class="signature-count"><%= signature_count(:default, petition.signature_count) %></p>
+<div class="signature-count">
+  <p class="signature-count-number"><%= signature_count(:default, petition.signature_count) %></p>
+  <div class="signature-count-graph" aria-hidden="true">
+    <span class="signature-count-current" style="width:
+      <% if petition.signature_count < Site.threshold_for_response %>
+        <%= number_to_percentage((petition.signature_count / Site.threshold_for_response.to_f) *100) %>
+      <% elsif petition.signature_count < Site.threshold_for_debate %>
+        <%= number_to_percentage((petition.signature_count / Site.threshold_for_debate.to_f) *100) %>
+      <% else %>
+        100%
+      <% end %>
+      "></span>
+  </div>
+  <p class="signature-count-goal">
+    <% if petition.signature_count < Site.threshold_for_response %>
+      <%= number_with_delimiter(Site.threshold_for_response) %>
+    <% else %>
+      <%= number_with_delimiter(Site.threshold_for_debate) %>
+    <% end %>
+  </p>
+</div>
```

Comment:
This is too much logic to go in a template - it should be extracted to a helper method and tested